straxen/get_corrections.py

```diff
@@ -91,6 +93,11 @@ def get_correction_from_cmt(run_id, conf):
             else:
                 return float(correction) # float elife, drift velocity, etc
         
+        elif model_conf in arrays_corrections:
+            np_correction = correction.reshape(correction.size)
+            np_correction = np_correction.astype(np.int16)  # not sure if straxen can handle dtype:object therefore specify dtype
+            return np_correction
```

Comment:
> I see Daniels concerns, but I'm not sure if for this specific case interpolation would be the best approach. Usually these changes are not gradual at all for the hitfinding.

Sorry, I do not understand this comment. For sure there should not be any interpolation between threshold values :D. Thresholds should always be step-like functions versus time. Where did I give the impression that I ask for interpolated values? Then I can clarify maybe.   

> Aaron is right that it might be hard to get this from the CMT collections. Would it make sense to just do this at the plugin level (i.e. an array and set the astype(np.int16) on the self.thresholds)

Sorry, I am missing the knowledge of our database here. But in my naive understanding I would have thought that you can store a simple string (e.g. i2 or f4 for integer2byte or float4bvyte etc.) along with your data. This would make things more general and in my opinion a bit neater. If there are any technical challenges I am unaware of forget what I said.

Reply:
Ok, we are on the same page then, it was this comment:
"We should store a long the correction values some metadata which explains how the correction should be interpolated between time stamps."

>Sorry, I am missing the knowledge of our database here. But in my naive understanding I would have thought that you can store a simple string (e.g. i2 or f4 for integer2byte or float4bvyte etc.) along with your data. This would make things more general and in my opinion a bit neater. If there are any technical challenges I am unaware of forget what I said.

Yeah this is possible, in fact, we do such a thing e.g. for the online-monitor:
https://github.com/AxFoundation/strax/blob/master/strax/storage/mongo.py#L247-L254

I don't have a particularly strong preference for either putting such info in the database or doing an `astype(np.int16)` on the `self.thresholds`, I can see benefits of both